### PR TITLE
Update CI builds for 11.0

### PR DESCRIPTION
**Pull request by jpwhite4**
This is Ryans changes https://github.com/ubccr/xdmod-supremm/pull/357 with the review comments addressed by me.

Note the mongo version here are quite old - these will need to be updated in a separate pull request - which I think we'll want to move the mongo into a separate docker instance.

### CODE REVIEW

File: xdmod-supremm.spec.in
Comment (aaronweeden) [inline xdmod-supremm.spec.in:1]: Does the changelog at the bottom of this file need to be updated?

Comment (jpwhite4) [reply xdmod-supremm.spec.in:1]: We usually do that when we update the changelog in preperation for the final release. See e.g. https://github.com/ubccr/xdmod-supremm/pull/275/files

Comment (aaronweeden) [reply xdmod-supremm.spec.in:1]: I don't see release 10.5.0 listed below.

Comment (jpwhite4) [reply xdmod-supremm.spec.in:1]: The is must have been forgotten about for the 10.5 release

Comment (jpwhite4) [reply xdmod-supremm.spec.in:1]: will need to be a pull request into 10.5 then a pull request to merge up to 11

**Review by aaronweeden — APPROVED** [review]

**Pull request merged**